### PR TITLE
lazarsoma-FTS-12

### PR DESCRIPTION
**Pull request by lazarsoma**

### CODE REVIEW

File: src/main/java/com/greenfoxacademy/levelup/repository/IBadgeLevelRepository.java [hunk 1]
```diff
@@ -1,9 +0,0 @@
-package com.greenfoxacademy.levelup.repository;
-
-import org.springframework.data.repository.CrudRepository;
-import org.springframework.stereotype.Repository;
-
-@Repository
-public interface IBadgeLevelRepository extends CrudRepository {
-
```
Comment (rozanagy) [inline src/main/java/com/greenfoxacademy/levelup/repository/IBadgeLevelRepository.java]: please handle the conflict

File: src/main/java/com/greenfoxacademy/levelup/repository/IBadgeRepository.java [hunk 2]
```diff
@@ -1,9 +0,0 @@
-package com.greenfoxacademy.levelup.repository;
-
-import org.springframework.data.repository.CrudRepository;
-import org.springframework.stereotype.Repository;
-
-@Repository
-public interface IBadgeRepository extends CrudRepository {
```
Comment (rozanagy) [inline src/main/java/com/greenfoxacademy/levelup/repository/IBadgeRepository.java]: pls handle the conflict

File: src/main/java/com/greenfoxacademy/levelup/repository/IPersonRepository.java [hunk 3]
```diff
@@ -1,9 +0,0 @@
-package com.greenfoxacademy.levelup.repository;
-
-import org.springframework.data.repository.CrudRepository;
-import org.springframework.stereotype.Repository;
-
-@Repository
-public interface IPersonRepository extends CrudRepository {
```
Comment (rozanagy) [inline src/main/java/com/greenfoxacademy/levelup/repository/IPersonRepository.java]: pls handle the conflict

**Review by rozanagy — COMMENTED** [review]
:)

**Review by hulfyback — DISMISSED** [review]

**Review by rozanagy — DISMISSED** [review]

**Review by rozanagy — APPROVED** [review]

**Review by hulfyback — APPROVED** [review]
OK!

**Pull request merged**